.github/workflows/build.yml

```diff
@@ -132,14 +129,69 @@ jobs:
         name: reactos-syms-msvc${{matrix.toolset}}-${{matrix.arch}}-${{matrix.config}}-${{github.sha}}
         path: build/msvc_pdb
 
+  build-msvc_win2019:
```

Comment:
Why the new builder?

Reply:
The windows-2019 one seems to be the only one with an VS installation that contains the v14.0 build tools. If you replace the `runs-on:` with windows-latest (equivalently, that jobs section is merged back with the previous `build-msvc` one), the VS installation on that windows-latest doesn't have the v14.0 build tools and you get the builders errors that we currently have in master, see example below:

![image](https://user-images.githubusercontent.com/1969829/163872501-6d750155-869c-47ea-9faa-0bfb252ace69.png)

Reply:
Ah so this one is split off from the other win builder that used to do all 3?

Reply:
Exactly. And I tried before to see whether, in the original `build-msvc` job, I could override the `runs-on` property (I tried by re-specifying a new `runs-on: windows-2019` in the matrix specification where the VS2015/v14.0 build-tools were specified), but without success. The job always went to the windows-latest runner...

Reply:
> override the `runs-on` property
> without success

Odd: `matrix.os` worked for me last time I used it...

Reply:
Do you have an example of that @SergeGautherie ?

Reply:
> example

https://github.com/SergeGautherie/reactos/commit/9435cb982f26b66d02aa3fdb99ff3392e27f70e3

Reply:
The `matrix.os` suggestion works great together with the `include`/`exclude` trick.